src/modlunky2/ui/play/play.py

```diff
@@ -163,14 +163,27 @@ def load_from_ini(self):
 
         for options in SECTIONS.values():
             for option in options:
-                self.options_frame.ini_options[option].set(getattr(self.ini, option))
+                option_var = self.options_frame.ini_options[option]
+                option_var.set(getattr(self.ini, option))
+                option_var.trace_add(
```

Comment:
Now the update happens through the trace of the variables. This was necessary for the int vars, but caused some problems to register too early. So registering here instead in order to also have this be consistent between int and bool vars.